### ISSUE TITLE
Multi select getting extra values

### ISSUE THREAD

**Issue by palviggi-flagbit** [akeneo/magento2-connector-community#164]
When importing a multiselect attribute, the condition for the join is too open, and it will get extra values with similar names.

### Environment and configuration

Akeneo connector 100.3.3 (but the code is the same on 100.4.0)

### Steps to reproduce

1. Have a multiselect attribute, with codes "diameter", "cable_diameter", "other"
2. Have a product with the attribute, with the values "cable_diameter" and "other" selected

### Expected result

1. Data imported as selected

### Actual result

1. After import, the product will have both diameter and cable_diameter selected

The error is in https://github.com/akeneo/magento2-connector-community/blob/v100.4.0/Job/Product.php#L994

the condition as written matches substrings, it should take the full value in consideration
```
IF ( locate(',', `top5_attributes`) > 0 ,
	`p`.`top5_attributes` like CONCAT('%,', `c1`.`code`, ',%')  --value in middle of list
	OR `p`.`top5_attributes` like CONCAT(`c1`.`code`, ',%') --at start of list
	OR `p`.`top5_attributes` like CONCAT('%,', `c1`.`code`)	, --at end of list
	`p`.`top5_attributes` = `c1`.`code` )
```

**Comment by thanninger** [comment]
We noticed the same issue.

**Comment by Dnd-Gimix** [comment]
Hi @thanninger @palviggi-flagbit ,

Thank you both for you interest in Akeneo Connector !

We have released a bug fix for this issue in the latest 100.4.4 version of the connector : https://github.com/akeneo/magento2-connector-community/releases/tag/v100.4.4

Could you please give it a try and tell us if it solved your issue ?
Make sure to check the release notes before updating your connector.

Many thanks, I wish you a good day !

Regards,

**Comment by Dnd-Gimix** [comment]
Hello,

Without answer, I'll close this issue.

Beast regards,
